sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/Security/Ucfg/UcfgBuilder_Instructions.cs

```diff
@@ -598,6 +598,168 @@ public void Foo()
             UcfgVerifier.VerifyInstructions(code, "Foo");
         }
 
+        [TestMethod]
+        public void Invocations_QualifiedNames_Methods()

```

Comment:
I've added comprehensive unit tests for methods, property gets and sets 
i.e.
* global methods
* static global methods
* single and multiple namespaces
* nested classes

I haven't added a test for inner functions as they are currently unsupported by the C# CFG.
I'll create a separate SONARSEC ticket for this - the UCFG analyzer throws if it encounters an inner method.